src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -463,17 +465,46 @@ private void filmSuchen2(String urlFilm_, String thema, String titel, long dauer
         private String determineHdFromNormal(String urlNormal) {
           String urlHd = "";
           
-          // für URLs, die auf /960-1.mp4 enden, prüfen ob eine URL auf /1280-1.mp4 existiert
-          if (urlNormal.endsWith(SUFFIX_URL_NORMAL)) {
-            String urlTemp = urlNormal.replaceAll(SUFFIX_URL_NORMAL, SUFFIX_URL_HD);
-            if (urlExists(urlTemp)) {
-              urlHd = urlTemp;
+          // Dateiname extrahieren
+          int indexLastSlash = urlNormal.lastIndexOf("/");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around '/' to use the faster "lastIndexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)